external/miniz/Kconfig

```diff
@@ -0,0 +1,10 @@
+#
+# For a description of the syntax of this configuration file,
+# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+
+config LIBMINIZ
+	bool "LZMA library"
+	default y if COMPRESSION_TYPE=2 && BUILD_PROTECTED=n
```

Comment:
is `BUILD_PROTECTED=n` right?

Reply:
Yes. This is similar logic to what I used for LZMA compression library.
If `flat build`, build and include in `libexternal.a`
If protected build, have to take care whether to build and include as part of libkernel.a or libexternal.c

Reply:
Oh. I remembered I asked this before. This affects only default (initial) value so that user could enable on protected build. Don't you need to use dependency instead of this?